dev-lua/lua-openssl/files/lua-openssl-0.4.1-Makefile.patch

```diff
@@ -0,0 +1,53 @@
+From: Jakub Jirutka <jakub@jirutka.cz>
+
+Do not override CFLAGS, LDFLAGS and CC.
+
+--- Makefile
++++ Makefile
+@@ -22,30 +22,30 @@ SYS := $(shell gcc -dumpmachine)
+ 
+ ifneq (, $(findstring linux, $(SYS)))
+ # Do linux things
+-LDFLAGS		    = -fPIC -lrt -ldl
++LDFLAGS		    += -fPIC -lrt -ldl
+ OPENSSL_LIBS	?= $(shell pkg-config openssl --libs) 
+ OPENSSL_CFLAGS	?= $(shell pkg-config openssl --cflags)

```

Comment:
I just noticed this... this will break cross compiling and maybe other things, because `pkg-config` is hardcoded.

The Makefile should do:
```
PKG_CONFIG ?= pkg-config
[...]
OPENSSL_CFLAGS	?= $(shell $(PKG_CONFIG) openssl --cflags)
[...]
```